app/models/ambassador.rb

```diff
@@ -0,0 +1,41 @@
+class Ambassador < SimpleDelegator
```

Comment:
Why not just put these methods in user?

I can't do `Ambassador.where(created_at: (Time.now - 1.week)..Time.now)` - or `Ambassador.last` - which is stuff I would expect to be able to do.

I think another option would be having it be a concern?

Reply:
Good call thanks, a decorator isn't quite right here. Refactored to inherited from User instead -- lmk what you think.

Reply:
Ooooh! Clever, yeah, if that works that's nifty!